Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -28,7 +28,7 @@ public AnalyzeCommand(IFileSystem fileSystem = null)
         {
             engine ??= RE2Regex.Instance;
 
-            var validators = new ValidatorsCache();
```

Comment:
>var validators = new ValidatorsCache(); [](http://example.com/codeflow?start=12&length=39)

Extending our ability to mock throughout the targeted code path.

Reply:
Excellent.

You can reduce this call to simply pass 'fileSystem', however.

The redundancy in 'fileSystem: fileSystem' doesn't encode any additional information. 

DRY :)